source/ada/lsp-ada_handlers.ads

```diff
@@ -255,6 +273,8 @@ private
       Completion_Snippets_Enabled : Boolean := False;
       --  True if the client supports completion snippets
 
+      Client_Capabilities : LSP.Messages.ClientCapabilities;
```

Comment:
We already keep `ClientCapabilities` in `Message_Handler.Client.capabilities`. Let's replace `Client_Capabilities` with a function, that returns `Self.Client.capabilities`?

Reply:
That's right, I didn't note that there was already a `Message_Handler.Client`. So yes,
I'll add this function.